### ISSUE TITLE
NVDA does not read Formula tags in PDF

### ISSUE THREAD

**Issue by PaulRayius** [nvaccess/nvda#12715]
### Steps to reproduce: 

1. Open the attached PDF (I used Adobe Acrobat Pro DC)
2. Read the document

### Actual behavior:
NVDA reads "Heading level one" and then the text followed by "here is a formula" (the text in the paragraph tag after the H1. 
 And then NVDA stops reading. The Formula tag is not announced, the Alt text assigned to the Formula tag is not read. 

### Expected behavior:
NVDA should announce the Formula tag (similar to how it says "Graphic" for a Figure tag) and then read the Alt text that's been assigned to the Formula tag. 

#### NVDA installed/portable/running from source: 
Installed

#### NVDA version:
2021.1beta5
#### Windows version:
Version	10.0.19042 Build 19042

#### Name and version of other software in use when reproducing the issue:
Adobe Acrobat Pro DC version 2021.005.20060
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I have not tried with other versions of NVDA but I have heard similar complaints from other users.  I do not, however, have access to which versions of NVDA they are using. 
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
After running the COM Registration Fixing Tool NVDA worked even worse and wouldn't read the PDF at all, beyond the title of the document (in the document's Properties). 
[Testing a Formula tag.pdf](https://github.com/nvaccess/nvda/files/6938684/Testing.a.Formula.tag.pdf)

**Comment by PaulRayius** [comment]
This is still an issue in Acrobat 2023.003.20244 - 64 bit.

**Comment by raebened** [comment]
I am experiencing the same problem. Is there any suggested workaround until this is fixed.

**Comment by PaulRayius** [comment]
> I am experiencing the same problem. Is there any suggested workaround until this is fixed.

My only suggestion would be to use a different tag - a Figure, if it's a screenshot of a formula, or a Span, for example - and then provide Alt text starting with some wording to indicate that it's a formula.
